storybook/src/components/Select/Select.docs.mdx

```diff
@@ -0,0 +1,36 @@
+import { Canvas, Controls, Markdown, Meta, Primary } from "@storybook/blocks";
+import * as SelectStories from "./Select.stories.tsx";
+import README from "../../../../packages/css/src/components/select/README.md?raw";
+
+<Meta of={SelectStories} />
+
+<Markdown>{README}</Markdown>
+
+# Default
+
+<Primary />
+
+<Controls />
+
+## Multiple
+
+With `multiple` enabled the attribute `size` is allowed to set the number of rows. Multi select is harder to use
+on desktop as they require the user to hold down the `Ctrl` or `Cmd` key while clicking on the options.
+You might need to describe this behavior to the user. It is recommended to use checkboxes instead.
+On the other hand, it is a good choice for mobile devices.
+
+<Canvas of={SelectStories.Multiple} />
```

Comment:
I'm not sure we should show an example of `multiple`. A single select is already tricky to use for users, a multi select is downright horrible on desktops. It's nice that it works better on mobile devices, but you can't really know if something is only going to be used on a mobile device (if you're not building apps).

What do you think @VincentSmedinga?

Reply:
It does work great on (some, most?) mobile devices, but I agree that usability on desktops is bad. So I’m afraid I must agree to remove this for now and rewrite this paragraph to a ‘Do not use’ guideline.

Reply:
Could you make a suggestion on a 'Do not use' policy without example?

Reply:
Maybe try it yourself first, and we can add onto it? [You can look at gov.uk for inspiration](https://design-system.service.gov.uk/components/select/#avoid-adding-functionality-to-allow-selecting-multiple-options).